src/OrchardCore/OrchardCore.Users.Core/Services/UserStore.cs

```diff
@@ -524,5 +531,111 @@ public async Task<IList<IUser>> GetUsersForClaimAsync(Claim claim, CancellationT
         }
 
         #endregion IUserClaimStore<IUser>
+
+        #region IUserAuthenticationTokenStore
+        public async Task<string> GetTokenAsync(IUser user, string loginProvider, string name, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (user == null)
+            {
+                throw new ArgumentNullException(nameof(user));
+            }
+
+            if (string.IsNullOrWhiteSpace(loginProvider))
```

Comment:
Technically, you should throw an `ArgumentException` instead of a `ArgumentNullException` here. Also, using `IsNullOrEmpty` is generally enough.

Reply:
@ArieGato Please take the changes into account.